tests/e2e/pageObjects/pages/Transactions/Receive.ts

```diff
@@ -0,0 +1,26 @@
+import { by, element } from 'detox';
+
+import actions from '../../../actions';
+
+const ReceiveCoins = () => {
+  const ReceiveCoinsMainScreen = () => ({
```

Comment:
I think it doesn't make sense to have `ReceiveCoinsMainScreen` inside `ReceiveCoins` when `ReceiveCoinsMainScreen` is the only screen here. I'd change `ReceiveCoins` to `ReceiveCoinsScreen` and move everything there

Reply:
I agree